fee_calculator/apps/api/urls.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-from django.conf.urls import url, include
-from django.urls import path
+from django.conf.urls import include
+from django.urls import path, re_path
```

Comment:
`include` appears to be available in the `django.urls`, which would mean you can change to this;

```suggestion
from django.urls import include, path, re_path
```

Do you know if there is any reason not to do this?

Reply:
Fixed 👍